### PR TITLE
Add apparmor enabled flag

### LINKED ISSUE #36: Add ability to disable apparmor profiles
**Describe what happened:**

Apparmor is not supported on EKS. When deploying the helm chart with `systemProbe` enabled, and PSPs enabled, even after fixing the issue described in #35 (by adding `unconfined` as an allowed profile), the pods fail to start ok EKS because apparmor profiles aren't supported. My pods show a state of `Blocked` and I see

```
Reason:               AppArmor
Message:              Cannot enforce AppArmor: AppArmor is not enabled on the host
```

in `kubectl describe`

I can't see an easy way of disabling them in the chart. Happy to provide a pull request to fix this. Is an `apparmorEnabled` flag possible?

**Describe what you expected:**

I should be able to deploy the Helm chart without modifying resources manually.

**Steps to reproduce the issue:**

Deploy the Helm chart with `datadog.systemProbe.enabled: true` and `agents.podSecurity.podSecurityPolicy.create: true` and `agents.podSecurity.apparmorProfiles: ["runtime/default", "unconfined"]`

**Additional environment details (Operating System, Cloud provider, etc):**

EKS, k8s 1.17

### PR DESCRIPTION
**Pull request by Niksko**
#### What this PR does / why we need it:

This adds a flag that allows users to control apparmor enforcement.
Some environments (in my specific case, Amazon Linux 2 on EKS) do not support apparmor profile enforcement. Without these changes, it's impossible to enforce PSPs on the Datadog agent pods. Using the chart as is results in pods in a `Blocked` state, because of the lack of ability to enforce apparmor.

#### Which issue this PR fixes
- fixes #36 

#### Checklist
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: charts/datadog/ci/disable-apparmor.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  appKey: "0000000000000000000000000000000000000000"
+  kubeStateMetricsEnabled: false
+  systemProbe:
+    enabled: true
+agents:
+  podSecurity:
+    podSecurityPolicy:
+      create: true
+  apparmor:
+    enabled: false
```
Comment (clamoriniere) [inline charts/datadog/ci/disable-apparmor.yaml:13]:
```suggestion
    enabled: false

```

**Review by clamoriniere — COMMENTED** [review]
Hi,

Thanks for this contribution, could you please look at my small comment.
Then I will by able to approve it

File: charts/datadog/values.yaml [hunk 2]
```diff
@@ -1130,6 +1130,12 @@ agents:
   #   # agent version >= 6.15
   #   # jmx_use_container_support: true
 
+  apparmor:
+    ## @param enabled - boolean - required
+    ## If true, enable apparmor enforcement, https://kubernetes.io/docs/tutorials/clusters/apparmor/
+    #
+    enabled: true
+
```
Comment (clamoriniere) [inline charts/datadog/values.yaml:1138]: sorry, I haven't saw it before, but could you move this new parameter inside `agents.podSecurity`

```yaml
agents:
  podSecurity:
     apparmor:
        enabled: true

```

Comment (Niksko) [reply charts/datadog/values.yaml]: Yep, no problem! Fixed.

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**